src/middlewared/middlewared/pytest/unit/plugins/test_port_attachments.py

```diff
@@ -0,0 +1,604 @@
+import contextlib
```

Comment:
I see no tests for IPv6 addresses (or an IPv6 wildcard)

Reply:
Yup on it already